### DISCUSSION TITLE
PHP 8.3.13 Entry Point Not Found Error

### DISCUSSION THREAD

**Discussion by HereToLearn101** [leokhoa/laragon#906] [Q&A]
I just installed the PHP 8.3.13 to Laragon on my Windows machine. When I start the service I get this error (screenshot below):

![Screenshot 2024-11-09 115905](https://github.com/user-attachments/assets/58094072-12bd-424d-b9ab-7b80c68102ee)

Has anybody experienced this error before, and what should I do to fix it?

**Comment by HereToLearn101** [comment]
I was originally using:
`https://windows.php.net/downloads/releases/php-8.3.13-Win32-vs16-x64.zip`
and this is the file that caused the error.

I installed:
`https://windows.php.net/downloads/releases/php-8.3.13-nts-Win32-vs16-x64.zip`
to test it... And this one did not give the error, and it seems to be working fine.

What's the difference? Sorry, I'm still very new to this. I've done some reading:

"TS refers to multithread capable builds. NTS refers to single thread only builds. Use case for TS binaries involves interaction with a multithreaded SAPI and PHP loaded as a module into a web server."

But doesn't make sense to me yet... For now, NTS is more compatible with Laragon than a non-NTS?

**Reply by luozhangyou** [reply]
this way can resolve it,i guess change apache verison also resove this error

**Comment by FlavioSuarez** [comment]
I saw this type of error when the OS version is not supported anymore...
Maybe your OS is too old?

**Reply by HereToLearn101** [reply]
I'll try this.

**Comment by yanuar113** [comment]
try updating apache to the latest version, I also got the same error then updated the apache version and the problem was resolved

![Screenshot 2024-11-11 223314](https://github.com/user-attachments/assets/745b3527-a635-4ec3-aac5-959feacd98bb)

**Reply by HereToLearn101** [reply]
I'll try this too.
